diff -p -up ./libgda-ui/demos/ddl_queries.c.strfmt ./libgda-ui/demos/ddl_queries.c
--- ./libgda-ui/demos/ddl_queries.c.strfmt	2012-04-28 15:44:50.000000000 +0000
+++ ./libgda-ui/demos/ddl_queries.c	2012-07-15 07:39:41.478967100 +0000
@@ -477,7 +477,7 @@ show_sql (G_GNUC_UNUSED GtkButton *butto
 
 		dlg = gtk_message_dialog_new_with_markup (GTK_WINDOW (data->top_window),
 							  GTK_DIALOG_MODAL,
-							  msg_type, GTK_BUTTONS_CLOSE, msg);
+							  msg_type, GTK_BUTTONS_CLOSE, "%s", msg);
 		g_free (sql);
 		g_free (msg);
 
diff -p -up ./tools/gda-sql.c.strfmt ./tools/gda-sql.c
--- ./tools/gda-sql.c.strfmt	2012-04-28 16:18:55.000000000 +0000
+++ ./tools/gda-sql.c	2012-07-15 07:39:41.487966681 +0000
@@ -6785,11 +6785,11 @@ void
 color_print (GdaSqlColor color, OutputFormat format, const char *fmt, ...)
 {
 	va_list args;
-	g_print (color_s (color, format));
+	g_print ("%s", color_s (color, format));
         va_start (args, fmt);
 	g_vprintf (fmt, args);
         va_end (args);
-	g_print (color_s (GDA_SQL_COLOR_RESET, format));
+	g_print ("%s", color_s (GDA_SQL_COLOR_RESET, format));
 }
 
 const gchar *
